app/Offer/OfferControllerProvider.php

```diff
@@ -47,6 +48,8 @@ public function connect(Application $app): ControllerCollection
         $controllers->put('/{offerType}/{offerId}/type/{termId}/', UpdateTypeRequestHandler::class);
         $controllers->put('/{offerType}/{offerId}/facilities/', UpdateFacilitiesRequestHandler::class);
 
+        $controllers->put('/{offerType}/{offerId}/priceInfo/', UpdatePriceInfoRequestHandler::class);
```

Comment:
Should be `/{offerType}/{offerId}/price-info/`, see https://docs.publiq.be/docs/api-design-guidelines/416a92442362d-rest#naming-conventions-for-uris